addons/calendar/models/calendar_attendee.py

```diff
@@ -110,7 +110,8 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
         ics_files = self.mapped('event_id')._get_ics_file()
 
         for attendee in self:
-            if attendee.email and attendee.partner_id != self.env.user.partner_id:
+            author = attendee.event_id.user_id.partner_id or self.env.user.partner_id
+            if attendee.email and (attendee.partner_id != author or self.env.context.get('mail_notify_author')):
```

Comment:
Does this change mean the line in appointment where we add "mail_notify_author" in the controller did not do anything? Or was it working through another mean (for example a mail.template sent by the tracking)? In that case are we receiving 2 emails?